src/utils.js

```diff
@@ -609,7 +609,7 @@ export function shuffle(array) {
 }
 
 export function deepClone(obj) {
-  return clone(obj);
+  return klona(obj) || {};
```

Comment:
Version 1 of just-clone gives back an empty object when the cloning fails.

The newer versions don't do that, just like klona.

This is here to preserve that behavior.